docs/changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 14-October-2020 - 17:49 CEST
+
+- [hotfix] Use non greedy regex to capture the pull-request number.
```

Comment:
😄

Reply:
🧑‍🏫 

This has been the PR that was failing: https://github.com/conan-io/conan-center-index/pull/3184, the title contains a `#<number>` that was captured instead of the PR number 😅